build_monitor/monitors.py

```diff
@@ -135,6 +135,14 @@ def update_state(self):
         repo = self.github.get_repo(self.config["github"]["user"] + "/" + self.config["github"]["repo"])
 
         dist_workflow = None
+        filename = ""
+
+        if self.tag_name.startswith("nightly_"):
+            filename = "build-nightly.yaml"
+        elif self.tag_name.startswith("release_"):
+            filename = "build-release.yaml"
+        else:
+            raise Exception("Invalid tag name. Not a \'release_\' or \'nightly_\'")
```

Comment:
In order to help find potentially malformed tag names, please add a call to `format()` to grab what the function was actually given.